beetsplug/lyrics.py

```diff
@@ -395,13 +395,15 @@ def fetch(self, artist, title):
 
         song_info = None
         for hit in json["response"]["hits"]:
-            if hit["result"]["primary_artist"]["name"] == artist:
+            if hit["result"]["primary_artist"]["name"].strip(u'\u200b').lower() == artist.lower():
                 song_info = hit
                 break
 
         if song_info:
             song_api_path = song_info["result"]["api_path"]
             return self.lyrics_from_song_api_path(song_api_path)
+        else:
+            self._log.debug(u'Genius did not return a matching artist entry')
```

Comment:
Suggest a shorter wording:
```suggestion
            self._log.debug(u'genius: No matching artist')
```